### PR TITLE
WT-8520 Reorder csuite tests in the long-test task

### PR DESCRIPTION
**Pull request by etienneptl**
In Evergreen, variables live within the scope of a task. If a function
defines a variable once, it will persist and be visible to the
subsequent function calls within that task. If a test should not use
extra variables, it should be executed before the extra variables are
defined.

### CODE REVIEW

**Comment by lukech** [comment]
> In Evergreen, variables live within the scope of a task. If a function
> defines a variable once, it will persist and be visible to the
> subsequent function calls within that task.

Thanks for digging it out. Though I believe this change will "work", it would be fragile for future extension (not everyone is aware of the special caveat). 

The variable used here is more of a task-level local variable. Not sure if Evergreen supports a function-level local variable. If not could be a good feature request.

**Review by lukech — APPROVED** [review]

**Comment by etienneptl** [comment]
> The variable used here is more of a task-level local variable. Not sure if Evergreen supports a function-level local variable. If not could be a good feature request.

@lukech, I agree. I have raised the issue with the Evergreen team. I am waiting for their answer before merging it, in case they have a better solution.

**Review by Siddhartha8899 — APPROVED** [review]

**Comment by etienneptl** [comment]
> > The variable used here is more of a task-level local variable. Not sure if Evergreen supports a function-level local variable. If not could be a good feature request.
> 
> @lukech, I agree. I have raised the issue with the Evergreen team. I am waiting for their answer before merging it, in case they have a better solution.

The reply from the Evergreen team was to effectively define the test that does need the argument first.

**Pull request merged**
